### PR TITLE
Implement local sound playback for common error handling #151

### PR DESCRIPTION
**Pull request by tommy5oo**

### CODE REVIEW

File: app/src/main/java/com/skt/nugu/sampleapp/widget/BottomSheetController.kt [hunk 1]
```diff
@@ -185,10 +185,26 @@ class BottomSheetController(
...
+            ASRAgentInterface.ErrorType.ERROR_NETWORK ,
+            ASRAgentInterface.ErrorType.ERROR_AUDIO_INPUT ,
+            ASRAgentInterface.ErrorType.ERROR_LISTENING_TIMEOUT -> {
+                bottomSheet.post {
+                    if (PreferenceHelper.enableRecognitionBeep(activity)) {
+                        SoundPoolCompat.play(SoundPoolCompat.LocalBeep.FAIL)
+                    }
+                }
+            }
+            ASRAgentInterface.ErrorType.ERROR_UNKNOWN -> {
```
Comment (shshshl09) [inline app/src/main/java/com/skt/nugu/sampleapp/widget/BottomSheetController.kt]: ERROR_NETWORK?

File: nugu-interface/src/main/java/com/skt/nugu/sdk/core/interfaces/capability/system/SystemAgentInterface.kt [hunk 2]
```diff
@@ -32,7 +32,7 @@ interface SystemAgentInterface {
         /**
          * After receiving an event, turn off immediately.
          */
-        fun onTurnOff()
+        fun onTurnOff() {/* default implementation */}
```
Comment (shshshl09) [inline nugu-interface/src/main/java/com/skt/nugu/sdk/core/interfaces/capability/system/SystemAgentInterface.kt]: Do not change default interface spec.
Recommend to use wrapper interface which implement this.

Comment (tommy5oo) [reply nugu-interface/src/main/java/com/skt/nugu/sdk/core/interfaces/capability/system/SystemAgentInterface.kt]: done

File: app/src/main/java/com/skt/nugu/sampleapp/widget/BottomSheetController.kt (hunk 1 above)
Comment (tommy5oo) [reply app/src/main/java/com/skt/nugu/sampleapp/widget/BottomSheetController.kt]: Perhaps in the case of ERROR_NETWORK, a connection error has already occurred or will occur.

**Pull request merged**